code/game/objects/items/devices/scanners.dm

```diff
@@ -28,9 +28,14 @@ MASS SPECTROMETER
 		START_PROCESSING(SSobj, src)
 
 /obj/item/device/t_scanner/proc/flick_sonar(obj/pipe)
-	var/image/I = image('icons/effects/effects.dmi', pipe, "blip", pipe.layer+1)

```

Comment:
Hmm, won't this stop working if the x-ray is in a backpack or dropped on the floor? That would be a problem imo.